[2025-04-08T23:14:22.851+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-08T23:14:22.872+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-08T23:11:20.261919+00:00 [queued]>
[2025-04-08T23:14:22.880+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-08T23:11:20.261919+00:00 [queued]>
[2025-04-08T23:14:22.881+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-04-08T23:14:22.921+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2025-04-08 23:11:20.261919+00:00
[2025-04-08T23:14:22.932+0000] {standard_task_runner.py:63} INFO - Started process 393 to run task
[2025-04-08T23:14:22.935+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'tmdb_pipeline_streamlit', 'load_to_postgres', 'manual__2025-04-08T23:11:20.261919+00:00', '--job-id', '894', '--raw', '--subdir', 'DAGS_FOLDER/tmdb_pipeline.py', '--cfg-path', '/tmp/tmpqo5k166r']
[2025-04-08T23:14:22.940+0000] {standard_task_runner.py:91} INFO - Job 894: Subtask load_to_postgres
[2025-04-08T23:14:22.993+0000] {task_command.py:426} INFO - Running <TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-08T23:11:20.261919+00:00 [running]> on host 4658e95262ac
[2025-04-08T23:14:23.097+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tmdb_pipeline_streamlit' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-04-08T23:11:20.261919+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-08T23:11:20.261919+00:00'
[2025-04-08T23:14:23.099+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-08T23:14:23.187+0000] {tmdb_pipeline.py:1859} ERROR - No se encontró el archivo de datos procesados: None
[2025-04-08T23:14:23.210+0000] {tmdb_pipeline.py:1869} INFO - Leyendo 20 registros procesados para cargar en PostgreSQL
[2025-04-08T23:14:23.217+0000] {tmdb_pipeline.py:1448} INFO - Tablas creadas o verificadas correctamente en PostgreSQL
[2025-04-08T23:14:23.221+0000] {tmdb_pipeline.py:1885} INFO - Conexión establecida con PostgreSQL
[2025-04-08T23:14:23.264+0000] {tmdb_pipeline.py:2038} INFO - Total de películas en la base de datos: 62
[2025-04-08T23:14:23.264+0000] {tmdb_pipeline.py:2039} INFO - Insertadas 0 nuevas películas en esta ejecución
[2025-04-08T23:14:23.265+0000] {tmdb_pipeline.py:2045} INFO - Datos cargados correctamente en PostgreSQL
[2025-04-08T23:14:23.266+0000] {python.py:237} INFO - Done. Returned value was: 20
[2025-04-08T23:14:23.266+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-08T23:14:23.288+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=tmdb_pipeline_streamlit, task_id=load_to_postgres, execution_date=20250408T231120, start_date=20250408T231422, end_date=20250408T231423
[2025-04-08T23:14:23.319+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-08T23:14:23.348+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-08T23:14:23.349+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
